docs/tutorial-model-and-deploy.md

```diff
@@ -1,7 +1,7 @@
 # Tutorial: Model and Deploy an Application with OpenTOSCA
 
 This tutorial takes you through the steps to model and deploy a simple application using OpenTOSCA in the development docker-compose setup.
-The application modeled in this tutorial is a simple Angular app hosted on NGINX running inside a docker container.
+The application modeled in this tutorial is the shor example that can also be found in the PlanQK marketplace.
```

Comment:
Maybe a link to the PlanQK marketplace if that's mentioned here?

Reply:
I have linked the homepage instead as the marketplace is behind the login.